transports/spvwallet/client_factory_adapter.go

```diff
@@ -65,28 +48,17 @@ func (bf *walletClientFactory) CreateWithXpriv(xpriv string) (users.UserWalletCl
 // CreateWithXpriv returns UserWalletClient as spv-wallet-go-client instance with given access key.
 func (bf *walletClientFactory) CreateWithAccessKey(accessKey string) (users.UserWalletClient, error) {
 	// Get env variables.
-	serverUrl, signRequest := getServerData()
+	serverUrl := getServerData()
 
-	userWalletClient, err := walletclient.New(
-		walletclient.WithAccessKey(accessKey),
-		walletclient.WithHTTP(serverUrl),
-		walletclient.WithSignRequest(signRequest),
-	)
-
-	if err != nil {
-		return nil, err
-	}
+	userWalletClient := walletclient.NewWithAccessKey(serverUrl, accessKey)
 
 	return &Client{
 		client: userWalletClient,
 		log:    bf.log,
 	}, nil
 }
 
-func getServerData() (serverUrl string, signRequest bool) {
+func getServerData() string {
 	// Get env variables.
-	serverUrl = viper.GetString(config.EnvServerUrl)
-	signRequest = viper.GetBool(config.EnvSignRequest)
```

Comment:
Since we don't use the config param `EnvSignRequest`, we should remove it from `viper.go` (and other places where it possibly occures)